docs/source/using/host-folders.adoc

```diff
@@ -36,22 +35,21 @@ This will be addressed in a future release of the minikube ISO.
 [[host-folder-prerequisite]]
 === Prerequisites
 
+==== SSHFS
+
+SSHFS is the default technology for sharing host folders. It works without any prerequisite.
+
 ==== CIFS
 
-To use the `minishift hostfolder` command for CIFS based host folders, you need to be able to share directories using CIFS.
-On Windows, CIFS is the default technology for sharing directories.
-For example, on Windows 10 the *_C:\Users_* directory is shared by default and can be accessed by locally-authenticated users.
+Host folders can be shared using CIFS.
+There is no prerequisite for using CIFS on Windows.
```

Comment:
I am not so sure we can say this. The pre-requisite is that it needs to be enabled, shares either need to be assigned, etc etc. It would be best to leave this statement out.